config.rb.sample

```diff
@@ -14,18 +14,20 @@ $new_discovery_url="https://discovery.etcd.io/new?size=#{$num_instances}"
 #  token = open($new_discovery_url).read
 #
 #  data = YAML.load(IO.readlines('user-data')[1..-1].join)
+#
 #  if data['coreos'].key? 'etcd'
 #    data['coreos']['etcd']['discovery'] = token
 #  end
+#
 #  if data['coreos'].key? 'etcd2'
 #    data['coreos']['etcd2']['discovery'] = token
 #  end
 #
 #  # Fix for YAML.load() converting reboot-strategy from 'off' to `false`
-#  if data['coreos']['update'].key? 'reboot-strategy'
-#     if data['coreos']['update']['reboot-strategy'] == false
-#          data['coreos']['update']['reboot-strategy'] = 'off'
-#       end
+#  if data['coreos'].key? 'update' and data['coreos']['update'].key? 'reboot-strategy'
```

Comment:
Can not see anything against that. And also now we can completely uncomment that code to allow automatic discovery substitution by default.